core/modules/installer/installer.authorize.inc

```diff
@@ -228,10 +228,27 @@ function installer_authorize_update_batch_finished($success, $results) {
       'type' => 'error',
     );
   }
-  // Since we're doing an update of existing code, always add a task for
-  // running update.php.
-  $results['tasks'][] = t('Your projects have been downloaded and updated.');
-  $results['tasks'][] = t('<a href="@update">Run database updates</a>', array('@update' => base_path() . 'core/update.php'));
+  $updates_needed = FALSE;
+  include_once(BACKDROP_ROOT . '/core/includes/install.inc');
+  foreach (module_list() as $module) {
+    $updates = backdrop_get_schema_versions($module);
```

Comment:
Testing this for `paragraphs` in `devel` 's Execute PHP.  Once the code update is done and the list of updates is shown on `core/update.php` - this function doesn't return anything.